app/models/github_repository.rb

```diff
@@ -161,7 +161,9 @@ def number_of_commits
   end
 
   def empty?
```

Comment:
Reviewers - this method is the only thing changed from the original PR

Reply:
Could we use the `GitHub::Errors` class here to follow the pattern in the other methods in the file? Not sure if there's anything different about this check but it could possibly look like this:

```
  def empty?
    GitHub::Errors.with_error_handling do
      @client.contents(full_name).empty?
    end
  rescue GitHub::Error
    return true
  end
```

Reply:
:+1:

Reply:
@srinjoym great suggestion!